C/include/utils.h

```diff
@@ -23,10 +30,22 @@ bool check_empty(char *str);
  */
 bool check_isbn(char *str);
 
+/**
+ * cette fonction renverse une chaine de caracteres
+ * @param str la chaîne de caractères à renverser
+ */
+char* new_strrev(char* str);
+
+/**
+ * cette fonction transforme un entier en une chaine de caracteres
+ * @param x l'entier a changer
+ */
+char* new_itoa(int x);
```

Comment:
This function already exists in stdlib. Plus i don't think you've implemented it
```suggestion
/**
 * cette fonction transforme un entier en une chaine de caracteres
 * @param x l'entier a changer
 */
char* new_itoa(int x);
```